packages/renderer/src/lib/image/RunImage.svelte

```diff
@@ -69,68 +170,661 @@ async function startContainer() {
+                    </button>
+                  </div>
+                {/each}
+                <label
+                  for="ContainerSecurityCapabilitiesDrop"
+                  class="pl-4 pt-2 block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
+                  >Drop from the container (CapDrop):</label>
+                <!-- Display the list of existing capDrop -->
+                {#each capDrops as capDrop, index}
+                  <div class="flex flex-row justify-center items-center w-full py-1">
+                    <input
+                      type="text"
+                      bind:value="{capDrop}"
+                      placeholder="Enter a kernel capability"
+                      class="ml-4 w-full p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400" />
+
+                    <button
+                      class="ml-2 p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400"
+                      hidden="{index === capDrops.length - 1}"
+                      on:click="{() => deleteCappDrop(index)}">
+                      <Fa class="h-4 w-4 text-xl" icon="{faMinusCircle}" />
+                    </button>
+                    <button
+                      class="ml-2 p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400"
+                      hidden="{index < capDrops.length - 1}"
+                      on:click="{addCapDrop}">
+                      <Fa class="h-4 w-4 text-xl" icon="{faPlusCircle}" />
+                    </button>
+                  </div>
+                {/each}
+
+                <!-- Specify user namespace-->
+                <label
+                  for="containerUserNamespace"
+                  class="pt-4 block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
+                  >Specify user namespace to use:</label>
+                <div class="flex flex-row justify-center items-center w-full">
+                  <input
+                    type="text"
+                    bind:value="{userNamespace}"
+                    placeholder="Enter a user namespace"
+                    class="ml-2 w-full p-2 outline-none text-sm bg-zinc-900 rounded-sm text-gray-400 placeholder-gray-400" />
+                </div>
+              </div>
+            </Route>
+
+            <Route path="/networking">
+              <div class="h-96 overflow-y-auto pr-4">
+                <!-- hostname-->
+                <label for="containerHostname" class="block mb-2 text-sm font-medium text-gray-900 dark:text-gray-300"
+                  >Defines container hostname:</label>
+                <div class="flex flex-row justify-center items-center w-full">
+                  <input
+                    type="text"
+                    bind:value="{hostname}"
+                    placeholder="The hostname to use for the container, as a valid RFC 1123 hostname."
```

Comment:
```suggestion
                    placeholder="Must be a valid RFC 1123 hostname"
```

We already describe "container hostname" in title, it's already implied were entering it.